packages/block-library/src/image/index.js

```diff
@@ -83,7 +89,7 @@ const schema = {
 		children: {
 			...imageSchema,
 			a: {
-				attributes: [ 'href' ],
+				attributes: [ 'href', 'target' ],
```

Comment:
Does `rel` need to be here?

Reply:
@aduth I'm honestly not completely sure but according to [this comment](https://github.com/WordPress/gutenberg/pull/9520#issuecomment-430961765), it seems the `rel` value gets set correctly when the HTML is copied over to a new paragraph element even without `rel` attribute added to the array. @talldan Any comments here?

Reply:
At the moment `rel="noreferrer noopener"` is added to any link with `target="_blank"` by the save function, which I think is a sensible default. I suppose some users might want a finer level of control over the attribute.

I think this is probably ok as a start, the code doesn't preclude more handling of `rel` being added later.

Reply:
It's not clear to me as someone implementing a block with a schema why the parser would disregard `rel`. I'm left confused why I should or shouldn't include the attribute here.

Reply:
It doesn't seem to be something mentioned in the handbook (nothing about raw transformations in general, as far as I could see), so potentially a gap in the information we provide.